internal/mongosh/mongosh_bin_windows.go

```diff
@@ -14,4 +14,18 @@
 
 package mongosh
 
+import (
+	"os"
+	"os/exec"
+)
+
 const mongoshBin = "mongosh.exe"
+
+func execCommand(args ...string) error {
+	cmd := exec.Command(mongoshBin, args...)
```

Comment:
[q] why not use `exec.Command` on every os?

Reply:
I'm assuming that the person who originally implemented this had a good reason to use `syscall.Exec` over `exec.Command`. So I opted to leave the existing code as is and implemented a working version for windows.

I can change it if we're positive that there was no special reason to use `syscall.Exec`

Reply:
I asked, there's no specific reason. Will update